core/src/transport/global_only.rs

```diff
@@ -292,20 +291,20 @@ impl<T: crate::Transport + Unpin> crate::Transport for Transport<T> {
         match addr.iter().next() {
             Some(Protocol::Ip4(a)) => {
                 if !ipv4_global::is_global(a) {
-                    debug!("Not dialing non global IP address {:?}.", a);
+                    tracing::debug!(ip=?a, "Not dialing non global IP address.");
                     return Err(TransportError::MultiaddrNotSupported(addr));
                 }
                 self.inner.dial(addr)
             }
             Some(Protocol::Ip6(a)) => {
                 if !ipv6_global::is_global(a) {
-                    debug!("Not dialing non global IP address {:?}.", a);
+                    tracing::debug!(ip=?a, "Not dialing non global IP address.");
                     return Err(TransportError::MultiaddrNotSupported(addr));
                 }
                 self.inner.dial(addr)
             }
             _ => {
-                debug!("Not dialing unsupported Multiaddress {:?}.", addr);
+                tracing::debug!(address=%addr, "Not dialing unsupported Multiaddress.");
```

Comment:
Let's remove dots at the end while we are at it for the log statements that we touch!